src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,37 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    private static final String CRYPT_ALGORITHM = "SHA-512";
+    private static final String HEX_FORMAT = "%02x";
+    private static final int DEFAULT_CAPACITY = 16;
+
+    public HashUtil() {
```

Comment:
this default constructor should be private according to common mistakes article
```suggestion
    private HashUtil() {
```